paths/electron/packaging-apps/outline.md

```diff
@@ -1,3 +1,20 @@
-## Learning goals
+# Electron 102: Packaging Electron Apps
 
-Take a locally running app, and package it up to be downloaded natively on other computers
+Main goal: Going from local to something someone can download
+
+## Packaging for different environments
+- Needs for Windows, OSX, and Linux
+- Packaging can be done for each OS, but the UI isn't automatically made to look native in the process
+
+## Tools to use
+- How to do this with Electron
```

Comment:
Is this "how to do this with Electron" section about making it have an OS native UI or to just package in general? Because we might be able to add that piece (making your app look native) as an extra credit section